pkg/services/notifications/mailer.go

```diff
@@ -27,12 +28,16 @@ func (ns *NotificationService) send(msg *Message) (int, error) {
 	}
 
 	var num int
-	for _, address := range msg.To {
-		m := gomail.NewMessage()
-		m.SetHeader("From", msg.From)
-		m.SetHeader("To", address)
-		m.SetHeader("Subject", msg.Subject)
 
+	h := make(map[string][]string)
```

Comment:
Might it make sense to factor out a common function for sending emails, to avoid the duplicated logic below for the two scenarios?